<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FIR_Designer</class>
 <widget class="QWidget" name="FIR_Designer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>454</width>
    <height>413</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Projektowanie filtrów FIR</string>
  </property>
  <layout class="QGridLayout" name="FIR_Designer_layout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <item row="2" column="0" colspan="3">
    <widget class="QSpinBox" name="numCoeffB">
     <property name="prefix">
      <string>Ilość współczynników: </string>
     </property>
     <property name="minimum">
      <number>5</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QSpinBox" name="srateB">
     <property name="suffix">
      <string> Hz</string>
     </property>
     <property name="prefix">
      <string>Częstotliwość próbkowania: </string>
     </property>
     <property name="minimum">
      <number>2</number>
     </property>
     <property name="maximum">
      <number>1000000000</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QPushButton" name="coeffGenB">
     <property name="text">
      <string>Wy&amp;generuj współczynniki filtru</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QSpinBox" name="precissionB">
     <property name="prefix">
      <string>Precyzja: </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>25</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QCheckBox" name="toClipboardB">
     <property name="text">
      <string>&amp;Kopiuj wygenerowane współczynniki do schowka</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>Typ filtru</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="0" column="0">
       <widget class="QRadioButton" name="lowPassB">
        <property name="text">
         <string>Dolnoprzepustowy</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QRadioButton" name="highPassB">
        <property name="text">
         <string>Górnoprzepustowy</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="8" column="0" colspan="3">
    <widget class="QGroupBox" name="textEditB">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string>&amp;Wprowadzaj wygenerowane współczynniki do pola tekstowego</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>3</number>
      </property>
      <property name="topMargin">
       <number>3</number>
      </property>
      <property name="rightMargin">
       <number>3</number>
      </property>
      <property name="bottomMargin">
       <number>3</number>
      </property>
      <property name="spacing">
       <number>3</number>
      </property>
      <item row="1" column="0" colspan="2">
       <widget class="QPushButton" name="textEditChooseB">
        <property name="text">
         <string>Wybierz &amp;pole tekstowe</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="textEditChoosedL">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Wybrane pole tekstowe:</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QSpinBox" name="cutoffB">
     <property name="suffix">
      <string> Hz</string>
     </property>
     <property name="prefix">
      <string>Częstotliwość graniczna: </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QPushButton" name="closeB">
     <property name="text">
      <string>&amp;Zamknij</string>
     </property>
     <property name="shortcut">
      <string>Esc</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="WindFunc" name="windFuncW" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WindFunc</class>
   <extends>QWidget</extends>
   <header>WindFunc.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>srateB</tabstop>
  <tabstop>cutoffB</tabstop>
  <tabstop>numCoeffB</tabstop>
  <tabstop>precissionB</tabstop>
  <tabstop>lowPassB</tabstop>
  <tabstop>highPassB</tabstop>
  <tabstop>toClipboardB</tabstop>
  <tabstop>textEditB</tabstop>
  <tabstop>textEditChooseB</tabstop>
  <tabstop>coeffGenB</tabstop>
  <tabstop>closeB</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>closeB</sender>
   <signal>clicked()</signal>
   <receiver>FIR_Designer</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>335</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>335</x>
     <y>300</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
